bbp/hpc/highfive/default.nix

```diff
@@ -11,12 +11,12 @@
 
 stdenv.mkDerivation rec {
   name = "highfive-${version}";
-  version = "1.4-dev201710";
+  version = "1.4-dev201805";
 
   src = fetchFromGitHub {
     owner = "BlueBrain";
     repo = "HighFive";
-    rev = "fce248203ca88c412fb32d5cb862628352d0bfe5";
+    rev = "6be97a0efcc618f1033a3745ff4030dc28cdd167";
```

Comment:
Do you need to bump highfive?

Reply:
We found together that the converter requires a recent HighFive on friday